node/package.json

```diff
@@ -27,12 +27,14 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-task-lib",
   "dependencies": {
+    "adm-zip": "^0.5.10",
+    "deasync": "^0.1.28",
     "minimatch": "3.0.5",
     "mockery": "^2.1.0",
+    "nodejs-file-downloader": "^4.11.1",
```

Comment:
Should we add it to the `ThirdPartyNotice.txt`?

Reply:
added